### DISCUSSION TITLE
Luke 1:28 "Hail, full of grace,"

### DISCUSSION THREAD

**Discussion by robertmcatee** [openenglishbible/Open-English-Bible#351] [Q&A]
What I want to ask is why you have chosen "highly favored" instead of "full of grace" for Luke 1:28?

The original Greek word used in Luke 1:28 is κεχαριτωμένη (charitoō), which can be translated as "highly favored" or "full of grace." The word "grace" in English has a range of meanings, including "favor," "kindness," and "mercy." The word "favor" in English can also mean "to be preferred over others."

In the context of Luke 1:28, the word "grace" is a better translation than "favor" because it captures the idea of God's unmerited favor. Mary was not favored by God because she was somehow better than other people. She was favored by God because of His love and mercy.

The word "favor" can also be interpreted as meaning that Mary was given special privileges or benefits by God. While it is true that Mary was given a special privilege, the primary focus of the angel's greeting is on God's grace. Mary was "full of grace" because she was receptive to God's grace. She was willing to let God work in her life and to use her for His purposes.

The word "grace" is also a better translation than "favor" because it is more consistent with the rest of the New Testament. The New Testament teaches that we are saved by grace through faith (Ephesians 2:8-9). This means that we are saved not because of anything we have done, but because of God's grace.

The angel's greeting to Mary in Luke 1:28 is a reminder that we are all recipients of God's grace. We are all favored by God because of His love and mercy. We are all "full of grace" because we are receptive to God's grace.

**Comment by Coburn** [comment]
Respectfully disagree.

I am a spoken-language interpreter, and we use different kinds of speech,
or "registers," in our daily work. There is the formal register, like court
or medical language, the informal register, in casual conversation, and the
intimate register, the slang words you use with family and friends.

A greeting is part of the informal or even the familiar register. "Full of
grace" is from the formal register. It is a non-idiomatic coinage. It is
"Biblish" or "translationese." Nobody talks that way in real life!
Furthermore, you are trying to make it theologically correct! (although I
would submit that even within the bounds of orthodoxy, I could take issue
with a few points of your theology). The angel was not being theologically
correct. He was greeting Mary.

I agree that "κεχαριτωμένη" is not a usual greeting. Mary was surprised by
it. It was creative use of language. The angel took the ordinary greeting,
"Grace!" and changed it. He said something like "Hey! Hello!
Super-already-graced-lady!" But it is a violation of the use of language to
fall back on the formal register in such a situation. We have to try to
translate it using something that sounds at least a little bit like a
greeting.

I don't have a good alternative to suggest. I might start with something
like this. If the ordinary greeting was "Hello, my dear," κεχαριτωμένη
would sound something like "Hello, my dearest of the dear!" It was an
adapted greeting, it had connotations of liking and appreciation, and it
was given a personal and superlative twist. It was said in a bit of an
"Amplified Version" sort of formation, according to the declension of the
word. THAT is why Mary was surprised.

I will stay out of the discussion from here on. I just wanted to give you
guys the benefit of my admittedly subjective perspective, in hopes that it
helps you come up with a wonderful translation.

Thanks,

Coby

**Reply by robertmcatee** [reply]
It's hard not to inadvertently impose theology on this one. That said, I only know of the Greek word and then the Latin "gratia plena" (grace in plenitude?). I have just never heard "highly favored" instead of "full of grace" in reference to Mary, which is what provoked my question. I really appreciate your comments. I found them helpful.

**Reply by foolishgrunt** [reply]
I'm late to this discussion, but out of curiosity, do you come from a Catholic background? Because to my admittedly Protestant ears, the phrase "full of grace" has a very Catholic ring - and not only on account of the Hail Mary prayer. The Bible translation I grew up reading (NIV) renders the angel's greeting in Luke 1:28 as "Greetings, you who are highly favored!" And nearly every translation I can remember picking up since then (NASB, NET, ESV, even ASV and KJV) has some permutation of the word "favor." The only translation I'm familiar with that renders the angel's reading as "full of grace" is Douay-Rheims.

That's not necessarily an argument against rendering it as "full of grace" - my mono-linguistic self certainly doesn't have the chops to lend anything of substance to the underlying question(s) of translation. I was just taken aback by your statement that you have never heard "highly favored" instead of "full of grace" in reference to Mary.

**Reply by robertmcatee** [reply]
It has been a while since I posed this question so I am not on this thought train anymore. That said, it is Sunday. What I recall discussing was that in Luke 1:28, the word that the angel uses is "kecharitomene". The Greek verb “to give grace” is "charitoo". This was my primary reasoning for grace being the best word as opposed to favor. Furthermore, full of grace has a long history and seems like the best choice. Protestant v. Catholic theology is out of scope. Also, I reject the "but the angel didn't speak Greek" argument. We have to go with what we have. What we have is a Greek text.

**Comment by Coburn** [comment]
Just a really minor point. Greek is the Bible's second language.
Nobody should ever base an important theological conclusion on the meaning
of a Greek word alone. In point of fact, the angel and Mary were probably
speaking Hebrew, or possibly Aramaic. For one thing, it would have been
quite undignified to announce the birth of the Jewish Messiah to his mother
in a Gentile language. I might hazard a guess that the angel said something
like רב שלומך *rab shlomekh*, "How great is your peace!" instead of *shalom
elayikh* or *mah shlomekh?*, "Peace!" or "How are you?" Just food for
thought.

**Reply by robertmcatee** [reply]
That's interesting, would you happen to know of any non Greek text that could be looked at for this phrasing (peace versus grace)?

**Comment by Coburn** [comment]
I hate doing a PPS but here goes...

See, we have to remember that κεχαριτωμένη was almost certainly a
*translation* of what the angel actually said!

**Reply by robertmcatee** [reply]
What is a PPS?

**Comment by Coburn** [comment]
To be clear, I am not referencing any specific non-Greek text. You could
check with the Jerusalem Perspective. They have done some preliminary
research on a possible underlying Hebrew *vorlage *to the Gospels. And I
have done a back-translation of 1 John into Hebrew, and I am very satisfied
with the results. But for this text, if there were any non-Greek text, I'm
sure you would have heard about it by now.

I am just saying that "Charis!" is a standard greeting in Greek, and
"Shalom!" is the equivalent in Hebrew. In one dialect of Chinese, I believe
it is "Have you eaten?" But this is a special register, a special class of
words that languages use. Stock words and phrases that you always use and
that never change. In English, we always, always, always say, "Hi. How are
you?" and the answer is always "Fine." And so for the angel to vary the
greeting--not change it entirely, just freely conjugate the word
differently--was very unusual. In our translations, we should reflect the
fact that this was a variation on a standard greeting. And my original
point was that "Hail, highly favored one!" and "Full of grace!" are not
standard greetings at all, nor are they variations of standard greetings.
They are Biblish, and we should rack our brains to come up with something
more natural-sounding, if we can.

And so, no, I am not basing my comment on anything that older and wiser men
have said. I am simply pointing out that this is a greeting, and so it
should sound like a greeting. Kind of a no-brainer, actually, but maybe
that's just me.

Thanks, guys.

PS a "PPS" is a "Post-post-script." A 3rd email in the chain. It's
embarrassing. But I felt like it was useful information. Thanks, and I've
enjoyed this discussion.

**Comment by Coburn** [comment]
I am relishing reflecting on this riddle.

Maybe it wouldn't work in the style of the translation, but what if the
angel said, "Hello! How fine, indeed!"

**Reply by robertmcatee** [reply]
I have been pondering this too. You gave me a lot of good points to think about. I appreciated your perspective on using something that is a greeting while also finding something that is more natural to the English Language. I am adding my own opinion by sticking to the word Grace but there are other words of esteem that could work. "Greetings" is a word that has been used for centuries to welcome someone. It is a word that is both formal and friendly. "Your Grace" is a title that is used to show respect for a high-ranking individual and it echos the original Latin translation "gratia" while omitting the non-English sounding "plena" or fullness. When you combine these two words, you create a greeting that is both respectful and welcoming. "Your Grace, greetings" or "Greetings, Your Grace!". I really like this one, because there is precedence for both the greeting and the title. If you wanted to embellish further, you could say "Greetings, Most Graceful!".

**Comment by Coburn** [comment]
Don't forget, this is the "incarnation," that is, the clothing of God in
flesh and bone. In other words, we need some very "human" clothing for this
phrase.

I don't want to overthink it. I'm just having fun with it.

**Comment by Coburn** [comment]
I've already said more than my share on this, but I would like to advocate
once again for using natural speech. My concern with both "highly favored"
and "full of grace" is that they're Biblish.

"If the whole church comes together and everyone speaks in tongues, and
enquirers or unbelievers come in, will they not say that you are out of
your mind? But if an unbeliever or an enquirer comes in while everyone is
prophesying, they are convicted of sin and are brought under judgment by
all, as the secrets of their hearts are laid bare. So they will fall down
and worship God, exclaiming, ‘God is really among you!’"

--1 Corinthians 14:23-25 NIV


I think the same thing is mildly true of using stilted, formal language and
overly-literal translations. People hear them and not only do they ask,
"What does that mean to me?" but they also ask, "Is it important that those
words are unnatural-sounding and formal? Is their awkwardness and formality
part of the message itself?" I think that since the discovery of Koiné
Greek, we would have to say that the awkward formality is definitely NOT
part of the message itself. The Bible is not written in any kind of awkward
or formal or even culturally biased language. Instead, the authors take
pains to be understood and to use words that their readers will find
familiar. I'm sure this is saying much too much on the subject, but people
don't seem to be getting it.

I would say that the angel did use a sort of awkward phrase in this
greeting, and the narrative makes a point of it. And so with that, I leave
you to your devices.

Best wishes,

Coby
